capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.google.univiz.api;
+
+import com.google.common.collect.Lists;
+import com.google.univiz.api.representation.CollegeId;
+import com.google.univiz.api.representation.SearchData;
+import com.google.univiz.api.representation.SuggestionResponse;
+import com.google.univiz.api.resource.SearchResource;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+
+final class SearchResourceImpl implements SearchResource {
+  private final SuggestionDataApi suggestionApi;
+
+  @Inject
+  public SearchResourceImpl(SuggestionDataApi suggestionApi) {
+    this.suggestionApi = suggestionApi;
+  }
+
+  @Override
+  public List<SearchData> getSearchSuggestions(String partialCollegeName) {
+    if (partialCollegeName == null || !partialCollegeName.matches("[a-zA-Z]+")) {
```

Comment:
What if the client is trying to complete "Boston University" and types "Boston Uni"? Won't this regex preclude that?

Reply:
Also, OOC - are we sure this is the right pattern to use? I can think of some counter examples:
* Texas A&M University
* University of California, Berkeley
* CO State-Pueblo
* University of Hawaiʻi at Mānoa 

... and so forth.

Reply:
is it better to remove the match statement altogether? seems like there are enough cases violating any reasonable regex that it doesn't make it work it